zeeguu_api/api/user_articles.py

```diff
@@ -33,6 +33,22 @@ def user_articles_starred_and_liked():
     return json_result(UserArticle.all_starred_and_liked_articles_of_user_info(flask.g.user))
 
 
+# ---------------------------------------------------------------------------
+@api.route("/cohort_articles/", methods=("GET",))
+@api.route("/cohort_articles/<int:count>", methods=("GET",))
+# ---------------------------------------------------------------------------
+@cross_domain
+@with_session
+def user_articles_cohort(count: int = 20):
+    """
+        get articles for the cohort associated with the user
+    """
+
+    return json_result(cohort_articles_for_user(flask.g.user, count))
+
+
+
+
```

Comment:
Too many blank lines, we should try to stick with the pep8 style guidelines, usually you have 2 blank lines between different methods.